octobot/backtesting/independent_backtesting.py

```diff
@@ -41,7 +41,8 @@ def __init__(self, config,
                  tentacles_setup_config,
                  backtesting_files,
                  data_file_path=backtesting_constants.BACKTESTING_FILE_PATH,
-                 run_on_common_part_only=True):
+                 run_on_common_part_only=True,
+                 disable_join_timeout=False):
```

Comment:
I think it would be clearer to name it "enable_timeout"

Reply:
Yes